syntax = "proto3";

package seerep;

message InstanceQuery
{
  // uuid of the project to query
  string projectuuid = 1;

  // uuid of the instance to query
  string instanceuuid = 2;
}
